server/api/contests.py

```diff
@@ -281,8 +281,11 @@ def validate_contests(contests: List[JSONDict], election: Election):
 # metadata.
 def set_contest_metadata(election: Election):
     for contest in election.contests:
-        if election.audit_type != AuditType.BALLOT_POLLING:
+        if election.audit_math_type == AuditMathType.CARDSTYLEDATA:
```

Comment:
Nit: Nest this under the ballot comparison case to clarify that this is one of the two options within that audit type